molecule/zabbix_proxy_psk/molecule.yml

```diff
@@ -0,0 +1,93 @@
+---
+dependency:
+  name: galaxy
+  enabled: true
+  options:
+    requirements-file: molecule/default/collections.yml
+
+driver:
+  name: default
+
+platforms:
+  - name: zabbix-proxy-psk-${MY_MOLECULE_VERSION:-v70}-${MY_MOLECULE_DATABASE:-mysql}-${MY_MOLECULE_CONTAINER:-rockylinux8}
+    image: geerlingguy/docker-${MY_MOLECULE_IMAGE:-rockylinux8}-ansible:latest
+    privileged: true
+    pre_build_image: true
+    command: ${MOLECULE_DOCKER_COMMAND:-""}
+    networks:
+      - name: zabbix
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+    cgroupns_mode: host
+    zabbix_proxy_version: ${MY_MOLECULE_VERSION:-v70}
+    groups:
+      - ${MY_MOLECULE_DATABASE:-mysql}
+      - ${MY_MOLECULE_VERSION:-v70}
+
+provisioner:
+  name: ansible
+  playbooks:
+    prepare: ../zabbix_proxy/prepare.yml
+    create: ../zabbix_proxy/create.docker.yml
+    converge: ../zabbix_proxy/converge.yml
+    destroy: ../zabbix_proxy/destroy.docker.yml
+  env:
+    # https://github.com/ansible/molecule/issues/4015#issuecomment-1680859724
+    ANSIBLE_ROLES_PATH: ../../roles
+  inventory:
+    group_vars:
+      all:
+        ansible_connection: community.docker.docker
+        zabbix_repo_apt_priority: 1
+        zabbix_repo_yum_gpg_check: 1
+        zabbix_api_create_proxy: true
+        zabbix_proxy_name: proxy1
```

Comment:
Are there scenarios where you'd want to give the proxy a different name than the `Hostname=` in zabbix_proxy.conf? (and does that scenario work? I can't seem to get that to work in my preliminary tests.)